module/applications/actor/trait-selector.mjs

```diff
@@ -94,18 +94,33 @@ export default class TraitSelector extends BaseConfigSheet {
   /** @inheritdoc */
   _getActorOverrides() {
     const overrides = super._getActorOverrides();
-    const path = Trait.changeKeyPath(this.trait);
-    const src = new Set(foundry.utils.getProperty(this.document._source, path));
-    const current = foundry.utils.getProperty(this.document, path);
-    const delta = current.difference(src);
-    for ( const choice of delta ) {
-      overrides.push(`choices.${choice}`);
-    }
+    const path = Trait.actorKeyPath(this.trait);
+    this.#addOverriddenChoices("choices", Trait.changeKeyPath(this.trait), overrides);
+    this.#addOverriddenChoices("bypasses", `${path}.bypasses`, overrides);
+    const pathCustom = `${path}.custom`;
+    const sourceCustom = foundry.utils.getProperty(this.document._source, pathCustom);
+    const currentCustom = foundry.utils.getProperty(this.document, pathCustom);
+    if ( sourceCustom !== currentCustom ) overrides.push(pathCustom);
     return overrides;
   }
 
   /* -------------------------------------------- */
 
+  /**
+   * Add choices that have been overridden.
+   * @param {string} prefix       The initial form prefix under which the choices are grouped.
+   * @param {string} path         Path in actor data.
+   * @param {string[]} overrides  The list of fields that are currently modified by Active Effects. *Will be mutated.*
+   */
+  #addOverriddenChoices(prefix, path, overrides) {
+    const source = new Set(foundry.utils.getProperty(this.document._source, path) ?? []);
+    const current = foundry.utils.getProperty(this.document, path) ?? new Set();
+    const delta = current.symmetricDifference(source);
```

Comment:
Was `difference`, but needs to `symmetricDifference`: an override my remove a choice.